### PR TITLE
Mcc materialize

### PR DESCRIPTION
**Pull request by Castoreno05**
Added a container with carousels. Commented out the tabs and cards to add the carousels. Moved the footer that was in the main and placed it below the body. Adjusted the search-data cards spacing.

### CODE REVIEW

**Review by littletonjeffrey — APPROVED** [review]
Nice job Matt.  Carousels look great, and the spacing of the elements is much cleaner

File: results.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <!--Import Google Icon Font-->
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <!--Import materialize.css-->
-  <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection" />
+  <!-- <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection" /> -->
```
Comment (codewizard-dt) [inline results.html:8]: Are you commenting out the Materialize CDN stylesheet?

File: results.html [hunk 2]
```diff
@@ -219,4 +290,30 @@ <h5 class="white-text">Links</h5>
   <script src="./assets/js/results.js"></script>
 </body>
 
+<!-- footer -->
```
Comment (codewizard-dt) [inline results.html:293]: I agree taking the `footer` out of `main` but it should still be inside the `body` tag according to [geeksforgeeks](https://www.geeksforgeeks.org/html5-footer-tag/)

**Review by codewizard-dt — APPROVED** [review]
Lookin' good.   A couple minor issues but nothing to keep from merging
Screen Shot 2022-07-10 at 8 10 06 PM
- Maybe a little padding on the header
- User input text is overlapping the icon
Screen Shot 2022-07-10 at 8 26 13 PM
- this is an example showing how to do initialize carousels two ways, with vanilla JS and with jQuery
- we shouldn't do both, so let's stick to jQuery since we are using it anyway

**Pull request merged**